projects/addon-editor/extensions/tiptap-node-view/node-view-render.ts

```diff
@@ -0,0 +1,157 @@
+import {DOCUMENT} from '@angular/common';
+import {Injector, Input, Type} from '@angular/core';
+import {
+    Editor,
+    NodeView,
+    NodeViewProps,
+    NodeViewRenderer,
+    NodeViewRendererOptions,
+} from '@tiptap/core';
+import type {Node as ProseMirrorNode} from 'prosemirror-model';
+import type {Decoration} from 'prosemirror-view';
+
+import {TuiComponentRenderer} from './component-render';
+
+/**
+ * You should extend this class to create custom
+ * Tiptap's {@link https://tiptap.dev/guide/node-views NodeView} from angular component.
+ * It contains compulsory properties which component will get externally while NodeView's rendering.
+ */
+export class TuiNodeViewNgComponent implements NodeViewProps {
+    @Input() editor!: NodeViewProps['editor'];
```

Comment:
Why are these marked as `@Input` if the whole class is undecorated anyway? Seems like this does nothing.